src/px-script/main/rangeslider/index.js

```diff
@@ -12,60 +14,58 @@ const _writeStyle = obj => {
     inlineStyle.textContent = styleText;
 };
 
-const rangeslider = (() => {
-    const init = () => {
-        const rangeContainers = document.querySelectorAll(".rangeslider");
-        const userBrowser = navigator.userAgent;
-        const inlineStyle = document.createElement("style");
-        const inlineStyleContent = [];
-        const isBrowserChrome = userBrowser.indexOf("Chrome") > -1;
-
-        if (rangeContainers.length > 0) {
-            if (isBrowserChrome) {
-                document.body.appendChild(inlineStyle);
-            }
+const init = ids => {
+    const rangeContainers = ids || ids === "" ? getElementsByIds(ids, "rangeslider") : document.querySelectorAll(".rangeslider");
+    const userBrowser = navigator.userAgent;
```

Comment:
Why create a const when it's only used 3 lines below to get a different variable? ⚰️